### PR TITLE
first pass at updating dtc

### PR DESCRIPTION
**Pull request by pstuffa**
Ready to merge!

### CODE REVIEW

File: docs/cells/data-table.md [hunk 1]
```diff
@@ -12,151 +10,51 @@ The **Data table cell** cell mode provides you with an easy way to put your data
   <figcaption>Observable's Data table cell.</figcaption>
 </figure>
 
-Once you have created a Data table cell, you can quickly sift through your data using common data manipulation operations:
-- [Filter](#filterSection) - Limit displayed results to matched criteria.
-- [Columns (Show/Hide)](#showHideSection) - Show or hide only certain columns.
-- [Sort](#sortSection) - Sort columns into ascending or descending order.
-- [Slice](#sliceSection) - Define a slice of data so that only limited rows are shown.
+There are multiple data wrangling options availble with the Data table cell:
```
Comment (eagereyes) [inline docs/cells/data-table.md:13]: Typo: "availble"

File: docs/cells/data-table.md [hunk 2]
```diff
@@ -12,151 +10,51 @@ The **Data table cell** cell mode provides you with an easy way to put your data
...
+- [Filter](#filter) - Limit displayed results to matched criteria.
+- [Columns (Show/Hide)](#show-and-hide-columns) - Show or hide only certain columns.
+- [Dervive columns](#create-new-columns-for-column-derivations) - create new columns derived from the original columns using JavaScript.
```
Comment (eagereyes) [inline docs/cells/data-table.md:16]: Typo: "Dervive"

File: docs/cells/data-table.md [hunk 3]
```diff
@@ -12,151 +10,51 @@ The **Data table cell** cell mode provides you with an easy way to put your data
...
-- An Apache Arrow or Parquet file
-- A database that you have connected (for instructions, see [Connecting to databases](https://observablehq.com/@observablehq/databases))
-
-To get started:
-1. Create the Data table cell from the cell menu.
-2. If necessary, prepare a data source by attaching a file or connecting to a database.
-3. Select the data source (and table, if using a database) from the Data table cell.
-
-## Creating a Data table cell
+## Add a Data table cell
```
[the remark this answers is not in the document]
Comment (pstuffa) [reply docs/cells/data-table.md:20]: The guidance I got was always "Data table cell" no matter the context, but I'm not a fan...

File: docs/cells/data-table.md [hunk 4]
```diff
@@ -676,20 +477,12 @@ A new SQL cell appears, with the same columns, filters, sorts, and slices:
...
-
-In combination with the Data table cell, you can make use of the [Databases Panel and Schema Viewer](https://observablehq.com/@observablehq/databases-panel-and-schema-viewer?collection=@observablehq/getting-data-in-and-out) to view a database's schema and insert cells pre-configured for a given database.
-
-## SQL mystery game / Data table cell tutorial
-
-If you're up for a challenge, the [SQL mystery game](https://observablehq.com/@observablehq/sql-mystery-game-data-table-tutorial?collection=@observablehq/sql-mystery-game) notebook asks you to solve a mystery using Data table cells. Take a look!
\ No newline at end of file
+## Additional info
+- If you want to refer to the results from your Data table cell in another cell, you will need to give the Data table cell a name.
+- You can also use a custom database client as a data source for a Data table cell. In order for a database client to be recognized as a valid data source, it must satisfy the requirements outlined in [DatabaseClient Specification](/data/databases/databaseclient-specification).
```
Comment (eagereyes) [inline docs/cells/data-table.md:488]: Again, not sure about capitalization here, why not just keep it lower case?

**Review by eagereyes — APPROVED** [review]
I just added a few comments about small typos, looks good otherwise!

**Pull request merged**